release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,42 @@
+## Version 1.2.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.0
+
+### Bug Fixes
+* Fix quoting and url-encoding ([#153](https://github.com/opensearch-project/dashboards-reports/pull/153))
+* Remove hard coded localhost when calling API ([#172](https://github.com/opensearch-project/dashboards-reports/pull/172))
+* Fix Report Creation after Report Definition Creation ([#196](https://github.com/opensearch-project/dashboards-reports/pull/196))
+* fix csv missing fields issue and empty csv on _source fields ([#206](https://github.com/opensearch-project/dashboards-reports/pull/206))
+* Revert backend paths to opendistro  ([#218](https://github.com/opensearch-project/dashboards-reports/pull/218))
+
+
+### Infrastructure
+* Integrate notifications backend ([#129](https://github.com/opensearch-project/dashboards-reports/pull/129))
```

Comment:
this is also part of adding notifications (which was then removed), could you also put in enhancements like others

Reply:
Yeah okay!